src/components/DailyStatistics/TransactionHistoryCard.tsx

```diff
@@ -25,6 +25,7 @@ const styles = StyleSheet.create({
     fontFamily: "brand-bold",
     fontSize: fontSize(0),
     marginBottom: size(3),
+    flexShrink: 1,
```

Comment:
thanks for fixing this.
just for good to test this scenario : how about the categoryName's  text length  increase ? 
is this `flexShrink : 1` will handle it ?

Reply:
@maksha19 yes, the text length doesn't matter, it'd always wrap itself in it's own "grid" and be pushed downwards to fit